File: compare_number.cpp
#include<iostream>
using namespace std;

int main(){
    int num1=0,num2 = 0, num3 = 0; //initailize variables
    cout<<"please input 3 number" << endl;
    cin >> num1 >> num2>>num3;

    if(num1>num2)
    {
        if( num1 > num3 ){
            cout<<"The biggest number is " << num1; // num1 > num2&num3
        }
        else{
            cout<<"The biggest number is " << num3; // num3 >= num1 > num2
        }
    }

    else
    {
        if( num2 > num3 ){
            cout<<"The biggest number is " << num2; // num2 >= num1 & num3
        }
        else{
            cout<<"The biggest number is " << num3; // num3 >= num2 >= num1
        }
    }

    return 0;
}

File: compare_number.kt
fun main(args:Array<String>){

  val num1 = 10
  val num2 = 5
  val num3 = 2
  val max = 0
  
  if(num1 > num2)
  {
    if(num1 > num3)
    {
      max = num1;
    }
  }
  if(num2 > num1)
  {
    if(num2 > num3)
    {
      max = num2;
    }
  }
  else
  {
    max = num3;
  }
  
  print("The max number is $max")
}
